_cards.md

```diff
@@ -240,8 +240,8 @@ Generate an address for a card.
 </aside>
 
 Parameter | Description
---------- | ----------------------------------------------------------------------------------------------
-network   | The address network. Possible values are: `bitcoin`, `ethereum` or `litecoin`.
+--------- | --------------------------------------------------------------------------------------------------------------------------------------
+network   | The address network. Possible values are: `bitcoin`, `bitcoin-cash`, `bitcoin-gold`, `dash`, `ethereum`, `litecoin`, and `xrp-ledger`.
```

Comment:
`anonymous` and `interledger` are also valid network values according to [this](https://github.com/uphold/backend/blob/a26a4f31d5607261fd9e9d03e60e350a47a94f71/apps/api/controllers/card-controller.js#L219)

```suggestion
network   | The address network. Possible values are: `anonymous`, `bitcoin`, `bitcoin-cash`, `bitcoin-gold`, `dash`, `ethereum`, `interledger`, `litecoin`, and `xrp-ledger`.
```

Reply:
Thanks for the heads-up about `interledger`, I'll add it. Upon additional internal discussion, we decided not to commit to support the `anonymous` pseudo-network by publicly documenting it (for the time being at least).